invenio_app_ils/config.py

```diff
@@ -776,6 +776,9 @@ def _(x):
 ILS_INDEXER_TASK_DELAY = timedelta(seconds=5)
 """Time delay that indexers spawning their asynchronous celery tasks."""
 
+ILS_SUPPORT_EMAIL = 'ils-support@test.org'
```

Comment:
there is already a config for this `SUPPORT_EMAIL`. Can you please remove?

Reply:
Yes